### PR TITLE
BuildResult Tooltip

### PR DESCRIPTION
**Pull request by DavidKang**
Using tooltips in buildresult are not every usable for small viewports,
even on a big viewport, due to it can contain a considerable amount on
the content.

#### project
Peek 2020-03-02 13-03

#### package
Peek 2020-03-02 13-03-2

### CODE REVIEW

File: src/api/app/views/webui/package/responsive_ux/_buildstatus.html.haml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          .d-flex.flex-row.flex-wrap.pt-1
+            .repository-state
+              - if !result.is_repository_in_db
+                %i.fas.fa-clock.text-warning{ title: 'This result is outdated' }
+              - else
+                = repository_status_icon(status: result.state)
+              %span.ml-1
+                = result.architecture
+            .build-state.toggle-build-info-parent
+              %i.fa.fa-question-circle.text-info.mr-1.toggle-build-info{ title: 'More information' }
```
Comment (hennevogel) [inline src/api/app/views/webui/package/responsive_ux/_buildstatus.html.haml]: I still find this click target rather small...

File: src/api/app/views/webui/package/responsive_ux/_buildstatus.html.haml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    %strong #{result.code}:
+                    %span= result.details
+
+        - previous_repo = result.repository
+    - else
+      All the results have state
+      %strong excluded/disabled
+
+:javascript
+  $('.toggle-build-info').on('click', function(){
```
Comment (krauselukas) [inline src/api/app/views/webui/package/responsive_ux/_buildstatus.html.haml]: @DavidKang this javascript code exists both in the ```build_results``` partials for package and project. Maybe you can put it into a function in the assets, and call it from there :)

Comment (krauselukas) [reply src/api/app/views/webui/package/responsive_ux/_buildstatus.html.haml]: Besides that the PR looks good to me

**Comment by DavidKang** [comment]
@hennevogel and @krauselukas, could you review again please?

**Pull request merged**